### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mberezhna**
- [DEMO LINK](https://mberezhna.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://mberezhna.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Nice start, but a lot of requirements need to be done.
Check your solution in accordance with the task and the checklist before creating a pull request. 
❗️ Copy this Checklist to the Pull Request description after links, and put - [x] before each point after you checked it.
image

**Comment by mberezhna** [comment]
- [DEMO LINK](https://mberezhna.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://mberezhna.github.io/layout_product-cards/report/html_report/)

- [+] there 2 BEM blocks `card` and `stars` each in it own file
- [+] SCSS Nesting is used for `elements`, `modifiers` and `pseudo-classes`
- [+] SCSS Variables are used for main values and placed in a **separate file**
- [+] all `stars--N` modifiers work as expected (Highlight first `N` stars)
- [+] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

File: src/styles/blocks/stars.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    background-image: url(../images/star.svg);
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(n+5) {
+    background-image: url(../images/star-active.svg);
+  }
```
Comment (And678) [inline src/styles/blocks/stars.scss:21]: Requirement not implemented:
> all stars--N modifiers work as expected (Highlight first N stars)

**Review by And678 — CHANGES_REQUESTED** [review]
Almost done, one requirement to fix :)

File: src/index.html [hunk 2]
```diff
@@ -6,8 +6,49 @@
...
+          <div class="card__priceTxt">Price:</div>
+          <div class="card__priceNumber">$2,199</div>
+        </div>
+
+        <a
+        href="#"
+        class="button button--txt"
+        data-qa="hover"
+        >buy
+        </a>
```
Comment (BudnikOleksii) [inline src/index.html:50]:
```suggestion
        <a
          href="#"
          class="button button--txt"
          data-qa="hover"
        >
          buy
        </a>
```
Don't forget about formatting

File: src/styles/blocks/stars.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```
Comment (BudnikOleksii) [inline src/styles/blocks/stars.scss:14]: Still not implemented, if I change the modifier it's still 4 active stars:
- all stars--N modifiers work as expected (Highlight first N stars)

File: src/styles/blocks/normalize.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
+$button-color: #00acdc;
+$button-activ: #fff;
```
Comment (BudnikOleksii) [inline src/styles/blocks/normalize.scss:2]: Create a separate file for variables and move them to it

File: src/index.html [hunk 5]
```diff
@@ -6,8 +6,49 @@
...
+        >
+      </a>
+
+      <div class="card__description">
+        <h2 class="card__productName">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__productCod">Product code: 195434</p>
+
+        <div class="card__marks">
+          <div class="stars">
```
Comment (BudnikOleksii) [inline src/index.html:29]: Active stars should change depending on the modifier. For example, if I add the `stars--3` class it should be 3 active stars

**Review by BudnikOleksii — CHANGES_REQUESTED** [review]
Almost done, check the comments

**Comment by mberezhna** [comment]
[DEMO LINK](https://mberezhna.github.io/layout_product-cards/)

[TEST REPORT LINK](https://mberezhna.github.io/layout_product-cards/report/html_report/)

[+] there 2 BEM blocks card and stars each in it own file

[+] SCSS Nesting is used for elements, modifiers and pseudo-classes

[+] SCSS Variables are used for main values and placed in a separate file

[+] all stars--N modifiers work as expected (Highlight first N stars)

[+] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,52 @@
...
+        <div class="card__price">
+          <p class="card__priceTxt">Price:</p>
+          <p class="card__priceNumber">$2,199</p>
+        </div>
+
+        <a
+        href="#"
+        class="button button--txt"
+        data-qa="hover"
+        >
```
Comment (BudnikOleksii) [inline src/index.html:50]:
```suggestion
        <a
          href="#"
          class="button button--txt"
          data-qa="hover"
        >
```
Don't forget about formatting

**Review by BudnikOleksii — APPROVED** [review]
Good job 👍

**Pull request closed without merging**